src/thirdparty/pythoncapi_compat.h

```diff
@@ -7,7 +7,10 @@
 // https://github.com/python/pythoncapi_compat
 //
 // Latest version:
-// https://raw.githubusercontent.com/python/pythoncapi_compat/master/pythoncapi_compat.h
+// https://raw.githubusercontent.com/python/pythoncapi-compat/main/pythoncapi_compat.h
+//
+// This file was vendored from the following commit:
+// https://github.com/python/pythoncapi-compat/commit/0041177c4f348c8952b4c8980b2c90856e61c7c7
```

Comment:
This commit appears to be already merged in the latest version of this file.

Reply:
This is what I wanted to convey in my comment. That the version of the file came from this specific commit.

Reply:
Ah, I see now, thanks for the explanation.